docs/en/observability/logs-stream.asciidoc

```diff
@@ -28,75 +28,62 @@ On your host, download and extract the installation package that corresponds wit
-. Set the name for your dataset data stream in the *Dataset name* field. The dataset name describes the data ingested and its structure. This field can contain anything that signifies the source of the data. The default value is `generic`.
-. Click *Advanced options* to open the *Integration settings*.
-+
---
-[role="screenshot"]
-image::images/custom-log-advanced-options.png[open integration settings]
---
-. Set the name for your namespace data stream in the *Namespace* field. The namespace is useful for grouping data such as an environment (dev, prod, or qa), a team, or a strategic business unit. Using a namespace makes it easier to search for data from a given source by using a matching pattern. The default value is `default`.
-. Click *Save and continue*.
-. Click *Copy to clipboard*, and paste the configuration in the `elastic-agent.yml` file located on your host where you extracted the {agent} installation package.
-. We recommend using an API key to grant the agent access to {es}. To create an API key for your agent, see {fleet-guide}/grant-access-to-elasticsearch.html#create-api-key-standalone-agent[Create API keys for standalone agents].
-. Replace `username: '${ES_USERNAME}'` and `password: '${ES_PASSWORD}'` with `api_key:` in the `elastic-agent.yml` file, and add the API key you created in the previous step. For example:
-+
 [source,yaml]
 ----
-[...]
 outputs:
   default:
     type: elasticsearch
-    hosts:
-      - 'https://da4e3a6298c14a6683e6064ebfve9ace.us-central1.gcp.cloud.es.io:443'
-    api_key: _Nj4oH0aWZVGqM7MGop8:349p_U1ERHyIc4Nm8_AYkw
-[...]
+    hosts: '<your-elasticsearch-url>:<port>'
+    api_key: 'your-api-key'
+inputs:
+  - id: your-log-id
+    type: filestream
+    streams:
+      - id: your-log-stream-id
+        data_stream:
+          dataset: generic
+        paths:
+          - /var/log/your-logs.log
 ----
-+
-NOTE: The format of the key is <id>:<key>. Make sure you selected *Beats* when you created your API key. Base64 encoded API keys are not currently supported in this configuration.
-
-[discrete]
-[[logs-stream-install-agent]]
-== Step 3: Install and start the {agent}
-With your configuration set, you're ready to install the {agent}. From the agent directory, run the the command that corresponds with your system to install the {agent} as a service. Do *not* enroll the agent in Fleet.
 
-NOTE: On macOS, Linux (tar package), and Windows, run the `install` command to
-install {agent} as a managed service and start the service. The DEB and RPM
-packages include a service unit for Linux systems with
-systemd, so just enable then start the service.
+To configure your {agent}, set the values for the fields in the example configuration:
 
-include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/run-standalone-widget.asciidoc[]
+. `hosts` – Your {es} URL and the port. On {ecloud}, this . 
+.. Copy the {es} endpoint and cluster ID and from your deployment's page.
```

Comment:
As discussed, only the endpoint should be needed. This will also simplify the text below.

Reply:
When I tested with just the endpoint, I wasn't able to ingest any logs data. I also tried with/without the port, and I was only able to ingest data with the URL in this format. Maybe I'm doing something wrong there?

Reply:
I tested this again, and was able to get it working with the endpoint URL if I had the port added, so we can simplify at least that far.

Reply:
@ruflin I used 443. I just tested again on a fresh deployment and got the same result. Is it possible I don't have something set up correctly that would require manually entering the port?